pkg/operator/operator/errors.go

```diff
@@ -51,6 +51,7 @@ const (
 	ErrNoAvailableNodeComputeLimit        = "operator.no_available_node_compute_limit"
 	ErrCortexPrefixedEnvVarNotAllowed     = "operator.cortex_prefixed_env_var_not_allowed"
 	ErrAPINotDeployed                     = "operator.api_not_deployed"
+	ErrInvalidOverriddenImage             = "operator.invalid_overridden_image"
```

Comment:
nit: `ErrDockerImageInaccessible` (and associated variables)

Reply:
Done!